examples/resources/awscc_eks_cluster/eks_cluster_encrypt_kubernetes_secrets.tf

```diff
@@ -0,0 +1,141 @@
+data "aws_partition" "current" {}
+
+data "aws_caller_identity" "current" {}
+
+locals {
+  policy_arn_prefix = "arn:${data.aws_partition.current.partition}:iam::aws:policy"
+  account_id        = data.aws_caller_identity.current.account_id
+}
+
+variable "eks_default_tags" {
+  description = "Default tags to be applied to EKS resources"
+  default = [
+    {
+      key   = "Environment"
+      value = "Development"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
+}
+
+variable "eks_cluster_name" {
+  description = "EKS Cluster Name"
+  type        = string
+  default     = "example-cluster"
+}
+
+variable "eks_cluster_version" {
+  description = "EKS Cluster Version"
+  type        = string
+  default     = "1.27"
+}
+
+variable "eks_cluster_subnets" {
+  description = "Subnets for EKS Cluster"
+  type        = list(string)
+  default     = ["subnet-xxxx", "subnet-yyyy"] // Provide a list of subnet-ids for Amazon EKS Cluster
+}
+
+resource "awscc_iam_role" "main" {
+  description = "IAM Role of EKS Cluster"
+  role_name   = "${var.eks_cluster_name}-role"
+  assume_role_policy_document = jsonencode({
+    Statement = [{
+      Action = "sts:AssumeRole"
+      Effect = "Allow"
+      Principal = {
+        Service = "eks.amazonaws.com"
+      }
+    }]
+    Version = "2012-10-17"
+  })
+  managed_policy_arns = [
+    "${local.policy_arn_prefix}/AmazonEKSClusterPolicy",
```

Comment:
remove the local and show the exact policy arn

Reply:
Removed and added exact policy arn